internal/envoy/v3/endpoint_test.go

```diff
@@ -32,6 +32,19 @@ func TestLBEndpoint(t *testing.T) {
 	protobuf.ExpectEqual(t, want, got)
 }
 
+func TestHealthCheckLBEndpoint(t *testing.T) {
+	got := HealthCheckLBEndpoint(SocketAddress("microsoft.com", 81), 8998)
+	want := &envoy_endpoint_v3.LbEndpoint{
+		HostIdentifier: &envoy_endpoint_v3.LbEndpoint_Endpoint{
+			Endpoint: &envoy_endpoint_v3.Endpoint{
+				Address:           SocketAddress("microsoft.com", 81),
+				HealthCheckConfig: HealthCheckConfig(8998),
```

Comment:
would prefer this not use a helper to fill in this field but rather explicitly write out the object we expect, just to ensure we're not using something under test to generate the expected value